pyspark/bigdl/models/rnn/README.md

```diff
@@ -103,6 +103,31 @@ PYSPARK_DRIVER_PYTHON=./venv/bin/python PYSPARK_PYTHON=./venv.zip/venv/bin/pytho
 * `--dampening` dampening for momentum, the default value is 0.
 * `--maxEpoch` max number of epochs to train, the default value is 30.
 
+#####In order to use MKL-DNN as the backend, you should:
+1. Define a model with Model(graph container)
```

Comment:
please explain what types of RNN model are support for mkldnn now

Reply:
Added.